### ISSUE TITLE
VDA5050 Module for FMS

### ISSUE THREAD

**Issue by UmangBR** [amosproj/amos2022ss03-turtlebot-fleet-management#39]
## User story
1. As a SD
2. I want to develop a VDA5050 module
3. So that I can create VDA5050 JSON strings and read VDA5050 packets to dict (including validation)

## Acceptance criteria
* Create valid and standardized packets

## Definition of done (DoD)
* Added only after week 5
* The same for all features
* Here goes the project specific part

## DoD general criteria
* Feature has been fully implemented
* Feature has been merged into the mainline
* All acceptance criteria were met
* Product owner approved features
* All tests are passing
* Developers agreed to release

**Comment by cat24max** [comment]
The VDA5050 module is done for creating packets. Reading packets into a dict is not yet possible.

**Comment by cat24max** [comment]
Validation of incoming packets will not be part of this issue
